users/enorris/authorized_keys

```diff
@@ -0,0 +1 @@
+ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIL0hqfaA3ZgGYI2EyalgJgjgPupIxvvhGU+jMlpaNeb4 enorris@rw.rs
```

Comment:
Good coloring

Reply:
That's bone. And the lettering is something called _Silian Rail_.